packages/devel/libevent/package.mk

```
@@ -0,0 +1,21 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+# Copyright (C) 2024-present Team LibreELEC (https://libreelec.tv)
+
+PKG_NAME="libevent"
+PKG_VERSION="2.1.12-stable"
```

```suggestion
PKG_VERSION="2.1.12"
```
otherwise our package updated gets problems

---

done